doc/user_guide/configuration/all-options.rst

```diff
@@ -1285,7 +1285,7 @@ Standard Checkers
 
 --spelling-dict
 """""""""""""""
-*Spelling dictionary name. Available dictionaries: en (aspell), en_AU (aspell), en_CA (aspell), en_GB (aspell), en_US (aspell).*
+*Spelling dictionary name. Available dictionaries: af (aspell), am (aspell), ar (aspell), ast (aspell), az (aspell), be (aspell), be_BY (aspell), be_SU (aspell), bg (aspell), bn (aspell), br (aspell), ca (aspell), cs (aspell), csb (aspell), cy (aspell), da (aspell), de (aspell), de_AT (aspell), de_CH (aspell), de_DE (aspell), el (aspell), en (aspell), en_AU (aspell), en_CA (aspell), en_GB (aspell), en_US (aspell), eo (aspell), es (aspell), es_ES (AppleSpell), et (aspell), fa (aspell), fi (aspell), fo (aspell), fr (aspell), fr_CH (aspell), fr_FR (aspell), fy (aspell), ga (aspell), gd (aspell), gl (aspell), gr (aspell), grc (aspell), gu (aspell), gv (aspell), he (aspell), hi (aspell), hil (aspell), hr (aspell), hsb (aspell), hu (aspell), hu_HU (AppleSpell), hus (aspell), hy (aspell), ia (aspell), id (aspell), it (aspell), it_IT (AppleSpell), kn (aspell), ku (aspell), ky (aspell), la (aspell), lt (aspell), lv (aspell), mg (aspell), mi (aspell), mk (aspell), ml (aspell), mn (aspell), mr (aspell), ms (aspell), mt (aspell), nds (aspell), nl (aspell), nl_NL (AppleSpell), nn (aspell), ny (aspell), or (aspell), pa (aspell), pl (aspell), pt_BR (aspell), pt_PT (aspell), qu (aspell), ro (aspell), ru (aspell), rw (aspell), sc (aspell), sk (aspell), sk_SK (aspell), sl (aspell), sr (aspell), srd (aspell), sv (aspell), sv_SE (AppleSpell), sw (aspell), ta (aspell), te (aspell), tet (aspell), tk (aspell), tl (aspell), tn (aspell), tr (aspell), uk (aspell), uz (aspell), vi (aspell), wa (aspell), yi (aspell), zu (aspell).*
```

Comment:
This is better than what we have but also I don't think it's exhaustive and there's enough dicts so that reading the doc one could think it is, and that this is all the dict we can handle (when there's probably even more). Something that we need to discuss in another MR and/or in #6736 imo
```suggestion
*Spelling dictionary name. Available dictionaries: en (aspell), en_AU (aspell), en_CA (aspell), en_GB (aspell), en_US (aspell).*
```

Reply:
Oh shit, didn't mean to commit this. But guess other changes are fin!

Reply:
Yeah this is going to create conflict in other MR where the doc was generated but whatever it needs to be done at some point. It's really annoying to have to reset this default spelling dict line each time I use tbump, so I have incentive to fix this by the way :D